mongodb/src/test/java/org/hibernate/ogm/datastore/mongodb/test/gridfs/Photo.java

```diff
@@ -33,7 +33,7 @@
 	@Lob
 	@GridFSBucket(BUCKET_NAME)
 	@BinaryStorage( GRID_FS )
-	private Blob content;
+	private Blob blobContent;
```

Comment:
Can you rename this as `contentAsBlob`, please?

Reply:
ok